src/tools/w3dviewer/utils.h

```diff
@@ -29,5 +29,6 @@ void InitializeSpinButton(CSpinButtonCtrl *spin, float value, float min, float m
 void UpdateEditCtrl(HWND hwnd, int delta);
 void DisableWindows(HWND window, bool disable);
 void SetWindowFloat(HWND hWnd, float f);
+float GetWindowFloat(HWND hWnd);

```

Comment:
const

Reply:
This can't be const since it's not a member function.